src/client/Client.cc

```diff
@@ -12678,6 +12678,18 @@ int Client::ll_getxattr(Inode *in, const char *name, void *value,
   return _getxattr(in, name, value, size, perms);
 }
 
+int Client::check_dummy_op(const UserPerm& perms)
```

Comment:
We should make an effort to not export test methods. Perhaps make a `class TestClient` in a new `TestClient.h` which inherits `Client`.

Reply:
`class TestClient` could even be part of the unit test source file instead.

Reply:
Oh, okay. I'll try it out